Assets/Scenes/TutorialLevel.unity

```diff
@@ -9545,7 +9557,7 @@ PrefabInstance:
       objectReference: {fileID: 0}
     - target: {fileID: 8827876707810031362, guid: 901e6b7f6ba08daf59f185b6868a2d12, type: 3}
       propertyPath: m_IsActive
-      value: 1
+      value: 0
```

Comment:
This disables the second Adam added in 19784d0

Reply:
Why even disable the second Adam, that character is used to show the working of the initiative system.

Reply:
Because we only have one main player in this level and it looks weird to see two of them.

Reply:
If we want to show this on the review, enable this in a specialized build, but imo don't commit it.

Reply:
I get that but what if we would like to show the initiative system work in demo, how will you demonstrate different orders in the players if you only have one player. Once that is shown we can delete the 2nd one imo.

Reply:
As it is only a alpha build of the game anyway, it does not matter if it contains "debug" objects like a second Adam.

Reply:
While it might be an alpha build, stuff that one only adds for a local demo shouldn't end up in github unless it is meant to be. I guess we can leave it enabled for now considering the review is Monday, but for the next time I'd rather not see these kinds of debug stuff in here. Is it fine if I remove the second Adam / disable it in the commit that starts the 0.5 development cycle?

Reply:
Yes but remove it not disable, as that avoids clutter in the scene file

Reply:
Got it, I’ve re-enabled it for now.